src/services/PostCache/findPostsInCache.ts

```diff
@@ -0,0 +1,56 @@
+import PostCache from "../redisCache";
+import type { InterfacePost } from "../../models";
+import { Types } from "mongoose";
+
+export async function findPostsInCache(
+  ids: string[] | any[]
```

Comment:
While is `ids` potentially an `any[]`?

Reply:
the `any[]` type is also not needed, removed it.